release-please-config.json

```diff
@@ -873,10 +873,10 @@
             ],
             "release-type": "python"
         },
-        "packages/google-cloud-cloudquotas": {
+        "packages/google-cloud-quotas": {
             "bump-minor-pre-major": true,
             "bump-patch-for-minor-pre-major": true,
-            "component": "google-cloud-cloudquotas",
+            "component": "google-cloud-quotas",
             "extra-files": [
                 "google/cloud/cloudquotas/gapic_version.py",
                 "google/cloud/cloudquotas_v1/gapic_version.py",
```

Comment:
we should update these lines.

Reply:
We cannot change the namespace as that will be a breaking change.  The following code would no longer work
```
from google.cloud import cloudquotas
from google.cloud import cloudquotas_v1
```